src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -326,10 +326,20 @@ public function prepare(Request $request): static
     /**
      * Sends HTTP headers.
      *
+     * @param null|positive-int $statusCode The status code to use. Override the statusCode property if set and not null.
```

Comment:
Might be worth clarifying that it won't actually set `$this->statusCode` but only be used for the call at hand

Reply:
My bad I read "overwrite", nevermind.